src/test/java/org/llorllale/cactoos/matchers/ScalarHasValueTest.java

```diff
@@ -29,7 +29,6 @@
 
 import org.cactoos.scalar.Constant;
```

Comment:
@borysfan Please, check this file too because it seems has a PDD too `@todo #81:30min Replace all uses of MatcherAssert.assertThat() with...`

Reply:
@fabriciofx I covered this PDD in my changes -> PDD removed.